blueprints/app/files/README.md

```diff
@@ -8,15 +8,16 @@ A short introduction of this app could easily go here.
 You will need the following things properly installed on your computer.
 
 * [Git](https://git-scm.com/)
-* [Node.js](https://nodejs.org/) (with NPM)
+* [Node.js](https://nodejs.org/)<% if (yarn) { %>
+* [Yarn](https://yarnpkg.com/)<% } else { %> (with NPM)<% } %>
 * [Ember CLI](https://ember-cli.com/)
 * [Google Chrome](https://google.com/chrome/)
 
 ## Installation
 
 * `git clone <repository-url>` this repository
 * `cd <%= name %>`
-* `npm install`
+* `<% if (yarn) { %>yarn<% } else { %>npm<% } %> install`
```

Comment:
nevermind, apparently that's already what you're doing... review on mobile is not a good idea :D